packages/block-editor/src/components/rich-text/content.js

```diff
@@ -1,3 +1,5 @@
+'use no memo';
+
```

Comment:
This directive disables memoization of the `valueToHTMLString` call in the `RichText.Content` component. The `Content` component is used in a `renderToString` call when serializing components to strings, and such components should never use hooks. Because `renderToString` has no dispatcher for hooks. But the compiler adds a `useMemo`.